forte/api/forte_python_module.cc

```diff
@@ -301,6 +306,15 @@ PYBIND11_MODULE(forte, m) {
     py::class_<DressedQuantity>(m, "DressedQuantity")
         .def("contract_with_rdms", &DressedQuantity::contract_with_rdms, "reference"_a,
              "Contract densities with quantity");
+
+    // export Embedding_density
+    py::class_<EMBEDDING_DENSITY>(m, "EMBEDDING_DENSITY")
+        .def(py::init<const std::map<StateInfo, std::vector<double>>&, 
+                      std::shared_ptr<SCFInfo>, std::shared_ptr<MOSpaceInfo>, 
+                      std::shared_ptr<ForteIntegrals>, std::shared_ptr<ForteOptions> >(), 
+                      "state_weights_map"_a, "scf_info"_a, "mo_space_info"_a, "ints"_a, "options"_a)
+        .def("rhf_rdms", &EMBEDDING_DENSITY::rhf_rdms, "RHF RDMs in the active space")
```

Comment:
As a point of style, say this _returns_ the RDMs in the docstrings.

Reply:
Ok, modified.